src/components/DataTable/_data-table.scss

```diff
@@ -1 +1,67 @@
 @import '~carbon-components/scss/components/data-table/data-table';
+
+.table-header-label {
+  &-start {
+    .bx--table-header-label {
+      width: 100%;
+      text-align: start;
+    }
+  }
+
+  &-center {
+    .bx--table-header-label {
+      width: 100%;
+      text-align: center;
+    }
+  }
+
+  &-end {
+    .bx--table-header-label {
+      width: 100%;
+      text-align: end;
+    }
+  }
+}
+
+.bx--data-table td:last-of-type {
+  padding-right: $spacing-09 + $spacing-02;
+}
+
+.bx--data-table td:first-of-type {
+  padding-left: $spacing-04 + $spacing-02;
+}
+
+.bx--table-sort__icon-unsorted,
+.bx--table-sort__icon {
+  margin: $spacing-03;
+}
+
+td.data-table {
+  &-start {
+    padding-right: $spacing-05 + $spacing-02;
+    padding-left: $spacing-05 + $spacing-02;
+  }
+
+  &-end {
+    padding-left: $spacing-05 + $spacing-02;
+    padding-right: $spacing-09 + $spacing-02;
+  }
+
+  padding-right: $spacing-04 + $spacing-02 !important;
+}
+
+html[dir='rtl'] {
+  td.data-table {
+    &-start {
+      padding-right: $spacing-05 + $spacing-02;
+      padding-left: $spacing-05 + $spacing-02;
+    }
+
+    &-end {
+      padding-left: $spacing-09 + $spacing-02;
+      padding-right: $spacing-05 + $spacing-02;
+    }
+
+    padding-right: $spacing-04 + $spacing-02 !important;
```

Comment:
please justify the use of `!important` - this shouldn't be necessary to apply the required styles atop Carbon's styling

Reply:
I have refactored all the padding changes out of this PR